src/nidigital/metadata/functions.py

```diff
@@ -0,0 +1,4665 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's add NI-provided (not the add-on) metadata to source control in its own PR.

Reply:
Why? It's all needed.

Reply:
At the end of the day it's all needed. But if you post the NI-provided metadata in one PR, the reviewer doesn't have to look at all those file after every diff. Consuming the metadata is more likely to require multiple rounds, and it's very helpful to not have the trivial stuff in the way.

Reply:
Sure, why not have another PR...